.github/workflows/node.js.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x, 20.x]
```

Comment:
Question: I saw the PR from https://github.com/adobe/asset-compute-integration-tests/pull/42. 

Doesn't this mean older node versions become unsupported?

Reply:
@tmathern removed node-14 and 16 which are EOL 1b68606ff1fbb34f2716460145f71885a79ecfb3
#42 was updated as well.